test/models/workarea/segment/rules/traffic_referrer_test.decorator

```diff
@@ -0,0 +1,10 @@
+require 'test_helper'
+
+module Workarea
+  decorate Segment::Rules::TrafficReferrerTest, with: :one_theme do
```

Comment:
I don't see how the theme could be affecting this but seems like we should fix in base?

Reply:
Yeah, I hit this at the end of the day when everyone was home already. I wanted to see if anyone else hit this issue before. I didn't know what else to do other than skip here. Let me know how you'd like me to resolve.

Reply:
Ok, let's do the fix in base - it should be written to handle `nil` situations anyways.